src/Random/Utility.h

```diff
@@ -0,0 +1,157 @@
+// -*- C++ -*-
+/***************************************************************************
+ *
+ * The IPPL Framework
+ *
+ * This program was prepared by PSI.
+ * All rights in the program are reserved by PSI.
+ * Neither PSI nor the author(s)
+ * makes any warranty, express or implied, or assumes any liability or
+ * responsibility for the use of this software
+ *
+ * Visit www.amas.web.psi for more details
+ *
+ ***************************************************************************/
+
+#ifndef IPPL_RANDOM_UTILITY_H
+#define IPPL_RANDOM_UTILITY_H
+
+#include <Kokkos_MathematicalConstants.hpp>
+#include <Kokkos_MathematicalFunctions.hpp>
+#include <Kokkos_Random.hpp>
+#include "Types/ViewTypes.h"
+
+namespace ippl {
+  namespace random {
+
+    /*!
+     * @struct randu
+     * @brief Functor to generate random numbers from a uniform distribution.
+     *
+     * This functor generates random numbers from a uniform distribution in the
+     * range [rmin, rmax].
+     *
+     * @tparam T Data type of the random numbers.
+     * @tparam GeneratorPool Type of the random number generator pool.
+     * @tparam Dim Dimensionality of the random numbers.
+    */
+    template <typename T, unsigned Dim>
+    struct randu {
```

Comment:
Why are these in Utility? Shouldn't be then in a file called UniformDistribution.h similar to NormalDistribution.h?